### PR TITLE
Adding sprng package

### PR DESCRIPTION
**Pull request by gmfricke**
Sprng is a distributed process-aware random number generator that avoids correlations in random number sequences across processes.

http://www.sprng.org/

### CODE REVIEW

File: var/spack/repos/builtin/packages/sprng/package.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sprng/package.py]: :point_up:

File: var/spack/repos/builtin/packages/sprng/package.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+
+    # A list of GitHub accounts to
+    # notify when the package is updated.
+    maintainers = ['gmfricke', 'patrickb314']
+
+    homepage = "https://www.sprng.org"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sprng/package.py]: I think it's just `http` :cry:

File: var/spack/repos/builtin/packages/sprng/package.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5', sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sprng/package.py]: Can you call the version `5.0` as it is on `sprng` web-site?

File: var/spack/repos/builtin/packages/sprng/package.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    variant('mpi', default=True, description='Enable MPI support')
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi', when='+mpi')
+    depends_on('fortran', when='+fortran')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sprng/package.py]: Does this actually compile for you? `c`, `cxx` and `fortran` are currently modeled differently from other languages and so we have no provider for them - they are assumed to be generally available. I think you need to remove this line for the package to work.

File: var/spack/repos/builtin/packages/sprng/package.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+
+    # A list of GitHub accounts to
+    # notify when the package is updated.
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sprng/package.py]: You can safely remove this comment

File: var/spack/repos/builtin/packages/sprng/package.py (hunk 3 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/sprng/package.py]: If you do, you'll need a version-specific URL or `url_for_version`

File: var/spack/repos/builtin/packages/sprng/package.py (hunk 4 above)
Comment (kayarre) [reply var/spack/repos/builtin/packages/sprng/package.py:41]: Can you help me understand the issue here? I would like to fix the issues, and make a different pull request as this one  appears to be stagnant. I think this is the only issue I am not sure how to fix.

**Pull request closed without merging**